src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -97,6 +107,7 @@ CaptureOptionsDialog::CaptureOptionsDialog(QWidget* parent)
 
   if (!absl::GetFlag(FLAGS_tracepoint_callstack_collection)) {
     ui_->threadStateChangeCallstackCollectionCheckBox->hide();
+    ui_->threadStateChangeCallstackMaxCopyRawStackSizeWidget->hide();
```

Comment:
Shouldn't this also be hidden/unclickable if the user doesn't click collect thread states?